backtrace-library/src/main/java/backtraceio/library/logger/BacktraceLogger.java

```diff
@@ -79,8 +79,8 @@ public static int e(String tag, String message, Throwable tr) {
      * @param level login level (debug, warn, error, off)
      *
      * @deprecated setting the logging level should be done directly in the passed custom logger
-     * implementation so as not to depend on the internal implementation of Backtrace internal
-     * LogLevel types.
+     * implementation so as not to depend on the internal implementation of
+     * Backtrace internal logger and used LogLevel types.
      *
```

Comment:
Maybe a little more immediate?

@deprecated Setting the logging level should now be done directly in the passed custom logger
implementation. LogLevel should not depend on the internal implementation of
Backtrace internal logger and used LogLevel types.

Reply:
done